cves/kernel/CVE-2019-13648.yml

```
@@ -393,8 +422,8 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: A proper fix may be to deny write access to the MSR register. However, if privileges are somehow elevated or security is evaded, this issue still stands. The official fix uses Defense in Depth to handle invalid state at the signal handling level, rather than the input level by verifying the status of HTM on the machine.
```

I would refine this, it is a little hard to follow. I would start with how this applies to DiD, then go from there.

Consider, but probably still modify:
```suggestion
    note: |
      The Defense in Depth strategy is applicable here as it provides multiple
      layers of security. The optimal solution restricts write access to the
      Machine-State Register (MSR). However, if privileges are escalated or
      security measures bypassed, this vulnerability persists. Hence, the
      official resolution employs Defense in Depth. It manages invalid states
      at the signal handling level, not at the input level, by validating the
      status of the Transactional Memory (TM) on the system.
```

---

Fixed